tests/baselines/reference/assignmentCompatWithCallSignatures4.errors.txt

```diff
@@ -3,8 +3,6 @@ tests/cases/conformance/types/typeRelationships/assignmentCompatibility/assignme
     Type '(arg2: { foo: number; }) => any' is not assignable to type '(arg2: Base) => Derived'.
       Types of parameters 'arg2' and 'arg2' are incompatible.
         Type '{ foo: number; }' is not assignable to type 'Base'.
-          Types of property 'foo' are incompatible.

```

Comment:
Are these losses primarily because of the covariant / contravariant reversal, or something else in the change? It might be good to swap them back, just to see if it makes the errors go away.